### PR TITLE
Header animations added

### PR DESCRIPTION
**Pull request by LayanJay**
issue #3 

- New package installed (Framer-motion)
- Rename logoData.ts to LogoData.ts

### CODE REVIEW

File: animations/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
```
Comment (sithumonline) [inline animations/index.js]: Don't leave first line as empty line

File: modules/HomePage/OurPartners.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import Container from '../../components/Layout/Container'
 import NextImage from '../../components/NextImage'
 
 // partners logos
-import { logoData } from '../../lib/logoData'
+import { logoData } from '../../lib/LogoData'
```
Comment (sithumonline) [inline modules/HomePage/OurPartners.tsx:5]: Is this naming current 🤔

**Review by sithumonline — COMMENTED** [review]
LGTM

File: modules/HomePage/OurPartners.tsx (hunk 2 above)
Comment (LayanJay) [reply modules/HomePage/OurPartners.tsx:5]: yep

**Review by sithumonline — APPROVED** [review]

**Pull request merged**